### PR TITLE
Added sum to the end of the roll

### PR DESCRIPTION
**Pull request by Ramsfield**

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -93,8 +93,14 @@
 async def roll(ctx, arg1=1, arg2=100):
     "You can specify the amount of type of dice with a space, else it will be a random num between 1-100"
     author = ctx.message.author
+    message = ""
+    summ = 0
+    for i in range(arg1):
+        num = randint(1, arg2)
+        summ += num
+        message += f"Roll {i}: {num}\n"
     message = ''.join([f"Roll {i}: {randint(1, arg2)}\n" for i in range(arg1)])
```
Comment (CodeMastr3) [inline bot.py]: Is this line still supposed to be here?

File: bot.py [hunk 2]
```diff
@@ -54,7 +54,7 @@
 
 @bot.command()
 async def myroles(ctx):
-    "Lists rolls of member that called this function"
+    "Lists roles of member that called this function"
```
Comment (CodeMastr3) [inline bot.py]: Nice catch

File: bot.py [hunk 3]
```diff
@@ -66,7 +66,7 @@
 
 @bot.command()
 async def serverroles(ctx):
-    "Lists rolls of the server"
+    "Lists roles of the server"
```
Comment (CodeMastr3) [inline bot.py]: Again nice catch

**Comment by Ramsfield** [comment]
Great call -- should be changed now

**Comment by CodeMastr3** [comment]
Wonderful

**Pull request merged**